rizon/assetlist.json

```diff
@@ -19,6 +19,7 @@
         "display": "atolo",
         "symbol": "ATOLO",
         "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/rizon/images/atolo.png",
           "svg": "https://raw.githubusercontent.com/rizon-world/rizon-assets/master/logo/symbol%26favicon.svg"
```

Comment:
Please change to "https://raw.githubusercontent.com/cosmos/chain-registry/master/rizon/images/atolo.svg" and also upload the svg to the images folder here as well